routers/auth_router.js

```diff
@@ -81,17 +86,20 @@ router
 
 router
     .post('/sign-in', async (req, res) => {
+        const { error } = userValidation(req.body);
```

Comment:
/sign-up and /sign-in should use their personal validations. 1 route - 1 validation method